simulation/tests/test_agents.py

```diff
@@ -241,42 +246,106 @@ def test_households_not_suitable_archetype_are_not_heat_pump_suitable(
 
         assert not unsuitable_archetype_household.is_heat_pump_suitable
 
+    @pytest.mark.parametrize("event_trigger", list(EventTrigger))
     def test_heat_pumps_not_in_heating_system_options_if_household_not_heat_pump_suitable(
         self,
+        event_trigger,
     ) -> None:
 
         unsuitable_household = household_factory(potential_epc=Epc.D)
         model = model_factory()
         assert not HEAT_PUMPS.intersection(
-            unsuitable_household.get_heating_system_options(model)
+            unsuitable_household.get_heating_system_options(
+                model, event_trigger=event_trigger
+            )
         )
 
+    @pytest.mark.parametrize("event_trigger", list(EventTrigger))
     def test_heat_pumps_not_in_heating_system_options_if_household_not_heat_pump_aware(
         self,
+        event_trigger,
     ) -> None:
 
         unaware_household = household_factory(is_heat_pump_aware=False)
         model = model_factory()
         assert not HEAT_PUMPS.intersection(
-            unaware_household.get_heating_system_options(model)
+            unaware_household.get_heating_system_options(
+                model, event_trigger=event_trigger
+            )
```

Comment:
I can see it's not something you added in this PR, so don't feel obliged to change it, but I had to think about what's going on here with `assert not` and a set. I think something more explicit like the following is clearer.

```
heating_system_options = unaware_household.get_heating_system_options(model, event_trigger)
assert heating_system_options.intersection(HEAT_PUMPS) == {}
```